Library/Homebrew/utils/github.rb

```diff
@@ -501,6 +501,24 @@ def check_for_duplicate_pull_requests(name, tap_remote_repo, state:, file:, args
     end
   end
 
+  def get_long_build_pull_requests_files(name, repo)
+    pull_requests = search_issues("org:#{name}", repo: repo, state: "open", label: "\"long build\"")
+    prs = pull_requests.map { |pull_request| pull_request["number"] }
+    files = []
+    prs.each do |pr|
+      pr_files = API.open_rest(url_to("repos", name, repo, "pulls", pr, "files"))
+      pr_files.each do |pr_file|
+        files.append({ "file" => pr_file["filename"], "pr" => pr })
```

Comment:
I feel like there's a more intuitive way to organise how we keep track of these files, instead of an `Array` of `Hash` maps.

Reply:
Maybe there some ruby niceness that can improve this. My goal was to find a working solution first. If anyone has a better idea I'm open to implement it :)